protocol/avdl/stellar1/local.avdl

```diff
@@ -93,9 +93,10 @@ protocol local {
     string    fromUsername;    // Optional. Filled if KEYBASE.
 
     union { null, AccountID } toAccountID; // Optional.
-    string    toAccountName; // Optional. Filled if toType = OWNACCOUNT
-    string    toUsername;    // Optional. Filled if KEYBASE.
-    string    toAssertion;   // Optional. Filled if SBS.
+    string    toAccountName;               // Optional. Filled if toType = OWNACCOUNT
+    string    toUsername;                  // Optional. Filled if KEYBASE.
+    string    toAssertion;                 // Optional. Filled if SBS.
```

Comment:
"Filled if SBS" is no longer accurate since a canceled relay to an SBS will be blanked.